From: Johannes Thumshirn <morbidrsa@googlemail.com>
Date: Tue, 14 Aug 2012 20:22:48 +0200
Subject: [041/106] staging: line6: variax.c: Eliminated remaining
 strict_stroul()s

commit 9291975d6647794710553630c2613b5658c81d3f upstream.

Eliminated remaining calls to strict_stroul() and replaced them with
strict_kstrtou8().

Signed-off-by: Johannes Thumshirn <morbidrsa@googlemail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/variax.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

Index: linux-3.2.46/drivers/staging/line6/variax.c
===================================================================
--- linux-3.2.46.orig/drivers/staging/line6/variax.c	2013-07-26 19:28:40.000000000 +0000
+++ linux-3.2.46/drivers/staging/line6/variax.c	2013-07-26 19:28:51.000000000 +0000
@@ -353,10 +353,10 @@
 {
 	struct usb_line6_variax *variax =
 	    usb_get_intfdata(to_usb_interface(dev));
-	unsigned long value;
+	u8 value;
 	int ret;
 
-	ret = strict_strtoul(buf, 10, &value);
+	ret = kstrtou8(buf, 10, &value);
 	if (ret)
 		return ret;
 
@@ -387,10 +387,10 @@
 {
 	struct usb_line6_variax *variax =
 	    usb_get_intfdata(to_usb_interface(dev));
-	unsigned long value;
+	u8 value;
 	int ret;
 
-	ret = strict_strtoul(buf, 10, &value);
+	ret = kstrtou8(buf, 10, &value);
 	if (ret)
 		return ret;
 
